test/unit/sql/where.test.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
-const Support   = require(__dirname + '/../support'),
-  DataTypes = require(__dirname + '/../../../lib/data-types'),
+const Support = require(__dirname+'/../support'),
+  DataTypes = require(__dirname+'/../../../lib/data-types'),
```

Comment:
please readd the spaces, this change is not related to this feature and it's inconsistent with the other parts of the codebase